code/datums/greyscale/json_configs/plasmaman_helmet_protective_worn.json

```diff
@@ -0,0 +1,59 @@
+{
+	"helmet": [
+		{
+			"type": "icon_state",
+			"icon_state": "protective_base",
+			"blend_mode": "overlay",
+			"color_ids": [ 1 ]
+		},
+		{
+			"type": "icon_state",
+			"icon_state": "protective_stripe",
+			"blend_mode": "overlay",
+            "color_ids": [ 2 ]
+		},
+        {
+            "type": "icon_state",
+            "icon_state": "protective_visor",
+            "blend_mode": "overlay"
+        }
+	],
+    "helmet-light": [
+		{
+			"type": "icon_state",
+			"icon_state": "protective_base",
+			"blend_mode": "overlay",
+			"color_ids": [ 1 ]
+		},
+		{
+			"type": "icon_state",
+			"icon_state": "protective_stripe",
+			"blend_mode": "overlay",
+            "color_ids": [ 2 ]
+		},
+        {
+            "type": "icon_state",
+            "icon_state": "protective_visor_light",
+            "blend_mode": "overlay"
+        }
+	],
+    "helmet-weld": [
```

Comment:
should be overlays if all these helmets look the same now instead of being 40 different cached copies especially the visor which was changed from being an overlay

Reply:
Gags don't work well with overlays

Reply:
figured it out